### PR TITLE
set pnep url to be blank when no forecast is available

### PR DESCRIPTION
**Pull request by kgraaf**
This solves the issue of erroneous forecast raster data being "left behind" when updated to a month for which there is no forecast

### CODE REVIEW

**Review by aaron-kaplan — CHANGES_REQUESTED** [review]
If possible, it would be nice to display a message over the map saying something like "no forecast available for issue month {date}."

Also a minor stylistic comment below.

**Comment by aaron-kaplan** [comment]
If possible, it would be nice to display a message over the map saying something like "no forecast available for issue month {date}."

File: fbfmaproom/fbfmaproom.py [hunk 1]
```diff
@@ -843,9 +843,13 @@ def pnep_tile_url_callback(target_year, issue_month_idx, freq, pathname, season_
...
     target_month0 = season_config["target_month"]
 
-    # Prime the cache before the thundering horde of tile requests
-    select_pnep(country_key, issue_month0, target_month0, target_year, freq).data_array
-    return f"{TILE_PFX}/pnep/{{z}}/{{x}}/{{y}}/{country_key}/{season_id}/{target_year}/{issue_month_idx}/{freq}"
+    try:
+        # Prime the cache before the thundering horde of tile requests
+        select_pnep(country_key, issue_month0, target_month0, target_year, freq).data_array
+        return f"{TILE_PFX}/pnep/{{z}}/{{x}}/{{y}}/{country_key}/{season_id}/{target_year}/{issue_month_idx}/{freq}"
+    except:
```
[the remark this answers is not in the document]
Comment (kgraaf) [reply fbfmaproom/fbfmaproom.py:850]: Oh, that's a good point

**Comment by aaron-kaplan** [comment]
I don't love having the message in the control bar. It's hard to notice. Would it be hard to overlay it on the map? That's where I'm looking when I'm looking for the missing forecast.

**Comment by aaron-kaplan** [comment]
If you do manage to overlay the message on the map, try to configure it so that it only appears if the forecast layer is toggled on (visible).

**Comment by kgraaf** [comment]
> I don't love having the message in the control bar. It's hard to notice. Would it be hard to overlay it on the map? That's where I'm looking when I'm looking for the missing forecast.

I can try and see

**Comment by kgraaf** [comment]
Should the error message be placed in a tooltip like the "No Data" message?

**Comment by aaron-kaplan** [comment]
The tooltip isn't visible on initial page load. If you want to make it appear immediately if there's no forecast available, that seems ok.

File: fbfmaproom/fbflayout.py [hunk 2]
```diff
@@ -358,6 +358,16 @@ def command_layout():
...
+            dbc.Alert(
+                "No forecast available for this month",
+                color="danger",
+                dismissable=True,
+                is_open=False,
+                id="forecast_warning",
+                style={
+                    "margin-bottom": "8px",
+                },
+            ),
```
Comment (aaron-kaplan) [inline fbfmaproom/fbflayout.py:370]: Does this block interaction until you dismiss it? If so, that seems too heavyweight. Can't we add a label in the center of the map? Maybe you could use a [tooltip](https://leafletjs.com/reference.html#tooltip) with `permanent=true`?

Comment (aaron-kaplan) [reply fbfmaproom/fbflayout.py:370]: It doesn't block interaction.

**Review by aaron-kaplan — APPROVED** [review]

**Pull request merged**